hightouch_plugin_v2/client/src/App.js

```diff
@@ -0,0 +1,94 @@
+import React, {useEffect, useState} from 'react';
+import { client, useConfig } from '@sigmacomputing/plugin';
+import { ChakraProvider, Button, Box } from '@chakra-ui/react';
+
+// ---- Sigma Config -----
+client.config.configureEditorPanel([
+  { name: "Hightouch API Token", type: "text"},
```

Comment:
consider making this a secure text entry with:
```suggestion
  { name: "Hightouch API Token", type: "text", secure: true },
```